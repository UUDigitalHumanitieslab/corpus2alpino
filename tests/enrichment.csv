@lemma,@begin,english,_test
waarom,,why,
niet,,not,
,0,,hello
,,,fallthrough
