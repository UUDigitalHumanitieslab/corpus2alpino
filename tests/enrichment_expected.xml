<alpino_ds version="1.6" id="example_lassy.xml">
    <parser cats="1" skips="0"/>
    <node begin="0" cat="top" end="3" id="0" rel="top" _test="hello">
        <node begin="0" cat="advp" end="2" his="normal" his_1="normal" id="1" rel="--" _test="hello">
            <node begin="0" end="1" frame="wh_adverb" id="2" lcat="advp" lemma="waarom" pos="adv" postag="BW()" pt="bw" rel="hd" root="waarom" sense="waarom" wh="ywh" word="waarom" english="why"/>
            <node begin="1" end="2" frame="adverb" id="3" lcat="advp" lemma="niet" pos="adv" postag="BW()" pt="bw" rel="mod" root="niet" sense="niet" word="niet" english="not"/>
        </node>
        <node begin="2" end="3" frame="punct(vraag)" his="normal" his_1="normal" id="4" lcat="punct" lemma="?" pos="punct" postag="LET()" pt="let" rel="--" root="?" sense="?" special="vraag" word="?"/>
    </node>
    <sentence sentid="1000">waarom niet ?</sentence>
    <metadata>
        <meta type="text" name="charencoding" value="UTF8"/>
        <meta type="text" name="speaker" value="JAC"/>
        <meta type="text" name="origutt" value="waarom niet?"/>
        <meta type="text" name="role" value="Mother"/>
        <meta type="text" name="name" value="Jacqueline"/>
        <meta type="text" name="language" value="nld"/>
        <meta type="text" name="corpus" value="VanKampen"/>
        <meta type="int" name="uttid" value="999"/>
    </metadata>
</alpino_ds>